tasks/main.yml

```diff
@@ -1,9 +1,17 @@
 ---
-- include: "debian.yml"
-  when: ansible_os_family == "Debian"
+- include: "install_debian.yml"
+  when: ansible_os_family == "Debian" and strongloop_install
 
-- include: "redhat.yml"
-  when: ansible_os_family == "RedHat"
+- include: "install_redhat.yml"
+  when: ansible_os_family == "RedHat" and strongloop_install
+
+#
+# Add the support for multi versions of the same code
+#
+- include: strongloop_deploy.yml
```

Comment:
you may want to "wrap" this `with_items` loop into a different file, so you can fully skip the deploy if strongloop_deploy is set to false.